_includes/configure-registry-json.md

```diff
@@ -2,63 +2,50 @@
 
 ## Create a registry.json file
 
-After you’ve successfully installed Docker Desktop, create a `registry.json`
-file. Before you create a `registry.json` file, ensure that the developer is a
-member of at least one organization in Docker Hub. If the `registry.json` file
-matches at least one organization the developer is a member of, they can sign
-into Docker Desktop, and then access all their organizations.
+When creating a `registry.json` file, ensure that the developer is a member of at least one organization in Docker Hub. If the `registry.json` file matches at least one organization the developer is a member of, they can sign in to Docker Desktop and access all their organizations.
 
 ### Windows
 
-On Windows, you must create a file at
-`C:\ProgramData\DockerDesktop\registry.json` with file permissions that ensure
-that the developer using Docker Desktop cannot remove or edit the file (that is,
-only the system administrator can write to the file). The file must be of type
-`JSON` and contain the name of the organization in the `allowedOrgs` key.
+On Windows, run the following command in a terminal to install Docker Desktop:
 
-To create your `registry.json` file on Windows:
+`"Docker Desktop Installer.exe" install`
 
-1. Open Windows PowerShell and select Run as Administrator.
-2. Type the following command `cd /ProgramData/DockerDesktop/`
-3. Type `notepad registry.json` and enter the name of the Docker Hub
-   organization that the developer belongs to in the `allowedOrgs` key and click
-   **Save**. For example:
+If you’re using PowerShell, you should run it as:
 
-    ```json
-    {
-        "allowedOrgs": ["myorg"]
-    }
-    ```
+`Start-Process '.\win\build\Docker Desktop Installer.exe' -Wait install`
+
+If using the Windows Command Prompt:
+
+`start /w "" "Docker Desktop Installer.exe" install`
```

Comment:
Same for these (perhaps a different prompt to differentiate (e.g. `C:\Users\Admin>`), e.g. something like:

    ```console
    C:\Users\Admin> start /w "" "Docker Desktop Installer.exe" install
    ```